src/libs/auth.ts

```diff
@@ -289,10 +304,13 @@ const tryLoadAuthToken = async (includeBillingScope = false, popUp = false): Pro
     };
     return errorState;
   });
-  if (loadedAuthTokenResponse instanceof User) {
+  if (loadedAuthTokenResponse !== null) {
+    if ('status' in loadedAuthTokenResponse) {
```

Comment:
It looks like this is trying to determine if the `loadedAuthTokenResponse` is an `OidcUser` or `AuthTokenErrorState` object.

I suspect this code would be simpler if we removed the `.catch` from the `authInstance.signinX` and handled the error with a `try/catch`.

```ts
try {
  const loadedAuthTokenResponse: OidcUser | null = await (popUp
    ? // returns Promise<OidcUser | null>, attempts to use the refresh token to get a new authToken
      authInstance.signinPopup(args)
    : authInstance.signinSilent(args);

  if (loadedAuthTokenResponse === null) {
    return ...
  }

  // loadedAuthTokenResponse is type User
  ...
} catch (err) {
  ...
  return 
}
```
} catch